services/painting-surface-data-simulator-service/tests/test_scheduler_service.py

```
@@ -0,0 +1,336 @@
+    """시뮬레이터 스케줄러 테스트"""
+
+    def setup_method(self):
+        """각 테스트 메서드 실행 전 설정"""
+        self.scheduler = SimulatorScheduler()
+
+    def teardown_method(self):
+        """각 테스트 메서드 실행 후 정리"""
+        if self.scheduler.is_running:
+            # 이벤트 루프가 실행 중일 때만 create_task 사용
+            try:
+                loop = asyncio.get_running_loop()
```

Ruff (0.12.2)

Local variable `loop` is assigned to but never used

Remove assignment to unused variable `loop`

(F841)